docs/_includes/primary-nav-items.html

```diff
@@ -1,4 +1,4 @@
-<ul>
+<ul= class="nav-links">
```

Comment:
```suggestion
<ul class="nav-links">
```

Reply:
Any suggestions for code formatter for html css javascript on vscode